src/main/resources/avro/variants.avdl

```diff
@@ -41,7 +41,30 @@ record VariantSetMetadata {
 }
 
 /**
-`Variant` and `CallSet` both belong to a `VariantSet`.
+Method used for variant calling
+*/
+record CallingMethod {
```

Comment:
I might be missing it, but this looks like an orphan object -- it's not referenced by any other objects or methods. Without that, I don't understand how it's meant to be used.